pysb/pathfinder.py

```diff
@@ -147,6 +150,11 @@ def get_path(prog_name):
                                      set_path.__name__))
 
     search_paths = path_conf['search_paths'][os.name]
+    if use_path:
+        search_paths = list(search_paths) + [
+            path.strip('"') for path in
```

Comment:
Why strip double-quote? Have you encountered this on some system? I think windows does support quoting directories with semicolons in them (it's a valid path character there), but this code won't handle that case anyway. Also, under POSIX systems it seems double-quote is not treated specially at all. (I tested in Bash / Linux to confirm this)

Reply:
I think this was added in the original code preemptively. Agree with your feedback - I'll remove it unless/until a specific use case crops up later.